.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+---
+name: ""
+about: ""
+---
+
+<!--
+
+!!! IMPORTANT !!!
+
+Before creating your issue:
+
+* **Have a question?** Find community resources at https://github.com/asyncapi/community/issues/1.
```

Comment:
Not sure if already discussed but you can include those in the template chooser so it looks much cleaner: https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Reply:
It is included. There's a "question" type of issue. This one is for those who consider choosing "Open a new blank issue". It's just so they think twice 😄